### DISCUSSION TITLE
can you add the deafen thing or you can only  server deafen

### DISCUSSION THREAD

**Discussion by suptrpls** [jagrosh/MusicBot#731] [Ideas]
so if you add the deafen thing that well be better then  doing server deafen it might be a good idea but like you said when you answer ed me it not work right ?

**Comment by jagrosh** [comment]
Did you read through the existing issues? It looks like #519 is already what you want.

**Reply by suptrpls** [reply]
but are you ever gonna add it

**Reply by alexysarts** [reply]
I think regardless of whether it helps save bandwidth or not, there are still two good reasons why you should have it deafened. 
1) Some people feel paranoid or unsafe whether that's valid or not, and they'll be more likely to join a vc with a bot if they can see the deafened icon next to it. So I'm asking you for this so that members can feel more comfortable joining the vc. 
2) When you have a line-up of bots that deafen, (all the other ones do), and yours is the only one without that deafen icon, it looks goofy. So yeah. It's also more aesthetically pleasing to do this. If you manage to find the time to do this, I appreciate it.

**Reply by MichailiK** [reply]
I've made a pull request, #1491 so the bot always deafens itself. It should get into the 0.4.0 release soon.
